dev/devicelab/README.md

```diff
@@ -188,20 +188,20 @@ engine build. `--local-engine` is required for A/B test.
 `--ab-result-file=filename` can be used to provide an alternate location to output
 the JSON results file (defaults to `ABresults#.json`). A single `#` character can be
 used to indicate where to insert a serial number if a file with that name already
-exists, otherwise the file will be overwritten.
+exists, otherwise, the file will be overwritten.
 
 A/B can run exactly one task. Multiple tasks are not supported.
 
 Example output:
 
 ```
-Score	Average A (noise)	Average B (noise)	Speed-up
-bench_card_infinite_scroll.canvaskit.drawFrameDuration.average	2900.20 (8.44%)	2426.70 (8.94%)	1.20x
-bench_card_infinite_scroll.canvaskit.totalUiFrame.average	4964.00 (6.29%)	4098.00 (8.03%)	1.21x
-draw_rect.canvaskit.windowRenderDuration.average	1959.45 (16.56%)	2286.65 (0.61%)	0.86x
-draw_rect.canvaskit.sceneBuildDuration.average	1969.45 (16.37%)	2294.90 (0.58%)	0.86x
-draw_rect.canvaskit.drawFrameDuration.average	5335.20 (17.59%)	6437.60 (0.59%)	0.83x
-draw_rect.canvaskit.totalUiFrame.average	6832.00 (13.16%)	7932.00 (0.34%)	0.86x
+Score Average A (noise) Average B (noise) Speed-up
```

Comment:
This is actual log output, please don't reformat.

Reply:
Okay 👍

Reply:
I tried to make it correct but seems to be unable to put back the same spaces, even by copying the old version